src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -174,5 +174,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\$(AssemblyName)$(TargetExt)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName>StrongName</StrongName>
+    </FilesToSign>
+  </ItemGroup>
```

Comment:
I'll just do it manually in the merge commit, it's easier. Should I remove all the <FilesToSign> items from all the .csproj?